docs/resources/virtual_environment_vm.md

```diff
@@ -408,6 +408,14 @@ output "ubuntu_vm_public_key" {
     - `floating` - (Optional) The floating memory in megabytes (defaults
         to `0`).
     - `shared` - (Optional) The shared memory in megabytes (defaults to `0`).
+    - `hugepages` - (Optional) Enable/disable hugepages memory (defaults to disable).
+        - `2` - 2MB hugepages.
+        - `1024` - 1GB hugepages.
+        - `any` - Any hugepages.
+    - `keep_hugepages` - (Optional) Keep hugepages memory after the VM is stopped (defaults
+        to `false`).
+
+    Settings `hugepages` and `keep_hugepages` are only allowed for `root@pam` authenticated user.
```

Comment:
It also requires `numa` to be enabled. We can add it to the doc an also to the schema, using `RequiredWith` field on `mkMemoryHugepages` entry.

Reply:
Thanks, I added additional requirements in the documentation and code. 👍